jscomp/test/UncurriedAlways.js

```diff
@@ -210,6 +210,43 @@ function hello2(y, f) {
   return f(y);
 }
 
+let ReverseApplication = {
+  hello1: hello1,
+  hello2: hello2
+};
+
+function f(a, b, c) {
+  return [
+    b(a),
+    c(a)
+  ];
+}
+
+function f2(a, b, c, d, e) {
+  let __ocaml_internal_obj = a(b);
```

Comment:
Never noticed `__ocaml_internal_obj` in the output before.
Maybe we should rename it?